### ISSUE TITLE
bug: YouTube - thumbnail autoplay preview causes ReturnYouTubeDislike to fetch

### ISSUE THREAD

**Issue by LisoUseInAIKyrios** [revanced/revanced-integrations#305]
### Type

Cosmetic

### Bug description

Both ReturnYouTubeDislike and SponsorBlock are fetching data when video thumbnails are played in the feed.

With SponsorBlock this correct (sponsors are skipped, and the overlay skip controls work correctly).

But with ReturnYouTubeDislike it's fetching data it does not need.  And after the video player is opened, it fetches the same data again.

### Steps to reproduce

- turn on debug
- ensure "Settings -> Playback in set to " is set to 'always on'
- on the YouTube homepage, let a thumbnail video start playing
- open the video thumbnail to view in the player
- view debug logs: ReturnYouTubeDislike is loading when the thumbnail videos are loading

### Relevant log output

```shell
(viewing thumbnail playback in the feed):
01-24 20:33:25.587 25224 25224 D revanced: ReturnYouTubeDislike: New video loaded: -UYgORr5Qhg
01-24 20:33:25.588 25224 25641 D revanced: ReturnYouTubeDislikeApi: Fetching votes for: -UYgORr5Qhg

(open same video thumbnail in the regular video player):
01-24 20:33:41.988 25224 25224 D revanced: ReturnYouTubeDislike: New video loaded: -UYgORr5Qhg
01-24 20:33:41.988 25224 25641 D revanced: ReturnYouTubeDislikeApi: Fetching votes for: -UYgORr5Qhg
```

**Comment by LisoUseInAIKyrios** [comment]
Is there any existing ReVanced code or methods that check if the regular (non thumbnail) video player is currently open?

That would very easily fix this.

**Comment by oSumAtrIX** [comment]
Yes, there is an enum from which the current player type can be inferred from. "Minimized" playbacks such as those from the thumbnail appear as "None". Unfortunately I don't remember which patch refered to it right now...

**Comment by LisoUseInAIKyrios** [comment]
It was very easy to fix, and it allowed removing the use of the shorts detection patch
